scripts/dependencies.json

```diff
@@ -63,7 +63,7 @@
   {
     "groupId": "org.sonarsource.iac",
     "artifactId": "sonar-iac-plugin",
-    "version": "1.22.0.7057",
+    "version": "1.23.0.7263",
```

Comment:
I have verified that 

- ARM issues are still detected in JSON files
- CloudFormation issues still in YAML files
- JS issues still in YAML files